src/preprocessing.py

```diff
@@ -0,0 +1,37 @@
+import nltk
+import pandas as pd
+import re
+from nltk.corpus import stopwords
+from ast import literal_eval
+
+nltk.download('stopwords')
+REPLACE_BY_SPACE_RE = re.compile('[/(){}\[\]\|@,;]')
+BAD_SYMBOLS_RE = re.compile('[^0-9a-z #+_]')
+STOPWORDS = set(stopwords.words('english'))
+
+def read_data(filename):
+	data = pd.read_csv(filename, sep='\t')
+	data['tags'] = data['tags'].apply(literal_eval)
+	return data
+
+def text_prepare(text):
+	"""
+		text: a string
+		
+		return: modified initial string
+	"""
+	text = text.lower() # lowercase text
+	text = re.sub(REPLACE_BY_SPACE_RE, " ", text) # replace REPLACE_BY_SPACE_RE symbols by space in text
```

Comment:
These comments are useless, I would delete them (the code speaks for itself and comments are hard to maintain)